### ISSUE TITLE
v1.99.6 doesn't compile on Windows (MSVC)

### ISSUE THREAD

**Issue by nono303** [icing/mod_md#116]
Hi Stefan,

v1.99.6 doesn't compile on Windows. I just made a dirty patch according to breaking change to have it built. 
Please have a look, I'm not sure it's correct ;)
```
--- "a/mod_md_os-1.99.6.c"
+++ "b/mod_md_os-1.99.6_win.c"
@@ -67,7 +67,11 @@ apr_status_t md_try_chown(const char *fname, unsigned int uid, int gid, apr_pool
 
 apr_status_t md_make_worker_accessible(const char *fname, apr_pool_t *p)
 {
+#ifdef WIN32
+	return APR_ENOTIMPL;
+#else 
     return md_try_chown(fname, ap_unixd_config.user_id, -1, p);
+#endif
 }
```

**Comment by icing** [comment]
Sorry about that. Should be fixed by applying your patch in v1.99.7

**Comment by nono303** [comment]
Thanks!
1.99.7 sounds good and i'll test it
`md:info mod_md.c(1128) pid:6524 tid:1084 [AH10071: mod_md (v1.99.7-git), initializing...]`
